proto/internal/uber/cadence/admin/v1/service.proto

```diff
@@ -268,6 +271,15 @@ message DescribeClusterResponse {
   map<string,shared.v1.PersistenceInfo> persistence_info = 3;
 }
 
+message CountDLQMessagesRequest {
+  bool forceFetch = 1;
+}
+
+message CountDLQMessagesResponse {
+  repeated shared.v1.HistoryDLQCountEntry history = 1;
+  int64 domain = 2;
```

Comment:
I don't understand this being an int, sorry?

Reply:
We have two broad types of replication:
- History (which is sharded) therefore have multiple entries.
- Domain. This is "global" and has entirely different replication mechanism. Also there only single bucked for DLQs, therefore just a single number here.

Do you think we should have a different structure/naming here?